src/domain/RacingGame.js

```diff
@@ -1,40 +1,26 @@
-import { generateRandomNumber } from "../utils/GeneratorNumber.js";
 import Car from "./Car.js";
 
-export function createRacingCars(name) {
-  let cars = [];
-  const carNames = name.split(",");
-  carNames.forEach((name) => {
-    const car = new Car(name.trim());
-    cars.push(car);
-  });
+class RacingGame {
+  #racingTry;
 
-  return cars;
-}
+  constructor(racingTry) {
+    this.#racingTry = racingTry;
+  }
 
-export function racing(cars) {
-  const PLAY_TRY = 5;
+  createRacingCars(name) {
+    return name.split(",").map((name) => new Car(name.trim()));
+  }
```

Comment:
```suggestion
  createRacingCars(names) {
    return names.split(",").map((name) => new Car(name.trim()));
  }
```
인자를 names로 받아야 덜 헷갈릴 것 같아요!

Reply:
그리고 this를 사용하는 구간이 없어서, static method 로 만들어도 무방해보이네요 ㅎㅎ